### DISCUSSION TITLE
2024-04-10 Weekly Meeting 154 Notes/Agenda

### DISCUSSION THREAD

**Discussion by indiebio** [omigroup/omigroup#460] [Weekly Meetings]
Who saw the eclipse?

Weekly Meeting 154 is happening on Wednesday, 10 April, 2024 from 9:00 AM PDT (17:00 UTC) - 10:00 AM PDT (18:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting in the NEW SERVER, in the omi-weekly-meeting channel of the [OMI Discord](https://discord.gg/jVaUteJyh7).
We regularly use the # omi-meeting-chat channel for text chat.

### Agenda
Check in, welcome
Fun finds!
Working groups check in
bot update? backlog issues: https://github.com/orgs/omigroup/projects/7/views/2

AI for good hackathon: https://www.eventbrite.com/e/chesapeake-ai-hackathon-2024-tickets-862582195567

drafts: member interview questions
motivation draft blog post - https://github.com/omigroup/media/discussions/88
(digital apprenticeships - https://github.com/omigroup/media/discussions/87)

last week's notes: https://github.com/omigroup/omigroup/discussions/458

**Comment by indiebio** [comment]
member interview questions:

- What aspects of the metaverse are most important to you?
- What got you into the metaverse?
- What would you like people to know that is not listed in your bio?
- What is your project about?
- What frustrations do you have in this space?
- What other relevant communities do you belong to? What do you do there?

Please contribute questions:

1. if you're being interviewed, what would you want to be asked?
2. if you think of interesting people and projects you would want to learn more about, what would you ask them?

**Reply by indiebio** [reply]
humbletim: Interview from the future!!! But how many years in the future... 10 years is not enough.
Getting away of the dampening effect of reality. A plasticity of playing with thoughts and perspectives
taking account of what happens, after what happens next.

**Reply by indiebio** [reply]
what's the single most profound impact the metaverse, or the pursuit of it, has had on your life?

**Comment by indiebio** [comment]
mrmetaverse: Hii I'm helping with an AI hackathon which will include Magick as a key tool for agent development. OMI protocols and XR based additions are of course welcome. It starts next Friday. The theme is AI for Good, and focuses on agent creation. If you'd like to participate virtually you can sign up here: https://www.eventbrite.com/e/chesapeake-ai-hackathon-2024-tickets-862582195567

**Comment by mrmetaverse** [comment]
what are some blogs we should start working on?

**Reply by indiebio** [reply]
see: drafts: member interview questions
motivation draft blog post - https://github.com/omigroup/media/discussions/88
(digital apprenticeships - https://github.com/omigroup/media/discussions/87)

**Reply by indiebio** [reply]
https://vxtwitter.com/dankvr/status/1776862119184515394

**Comment by indiebio** [comment]
ECLIPSE!
Keyframe had a Texas festival but was closed a day early for tornado warnings, much apocalyptic.
SoloSolipsist had a great experience in Montreal

indiebio had partial eclipse, good enough for a beer (proof in the chunk missing in the green screen glare dot)
![image](https://github.com/omigroup/omigroup/assets/23474368/b943be0e-d170-4cf2-853f-3e87cbc007b7)

**Comment by indiebio** [comment]
Working group updates:
Publicity team: jimmy been busy
glTF doing gravity stuff
MSF, avatar wearables. new Chair, more members needed. Currently about voxel interoperability. Need a clearer value statement to trade off opportunity cost, what do people get for attending?

**Comment by indiebio** [comment]
M3 demo day soon - jin to post link
